src/prescriptions/prescription-tab-lists.component.tsx

```diff
@@ -167,11 +167,12 @@ const PrescriptionTabLists: React.FC = () => {
                                   <TableExpandedRow
                                     colSpan={headers.length + 1}
                                   >
-                                    <OrderExpanded
+                                    <PrescriptionExpanded
                                       encounterUuid={row.id}
                                       patientUuid={
                                         encounterToPatientMap[row.id]
                                       }
+                                      mutatePrescriptionTableRows={mutate}
```

Comment:
I had to pass this mutate function from this tab list component, into the prescription expanded component, and then into the dispense form component... it seemed a little "ugly" to have to pass this mutate function around so much, if we have to do this further we might want to refactor to use a context.  @pirupius  I'm interested in your thoughts about the best way to do this.  The issue is we generally don't have access to the current page, nor the search string, of the rows currently being displayed.

Reply:
context might be a better alternative but this is something i might need to first pull down and test.

Reply:
yeah, I didn't want to go overboard and add in a context if it wasn't really needed, as I have little experience with it.